3_Repositories.Tests/DataGenerators/HighlightDataGenerator.cs

```diff
@@ -0,0 +1,23 @@
+using Bogus;
+using Models;
+using Repositories.Tests.DataGenerators;
+using Repositories.Tests.DataGenerators.Base;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Repositories.Tests.DataGenerators
+{
+    public class HighlightDataGenerator : FakeDataGenerator<Highlight>
+    {
+        public HighlightDataGenerator()
+        {
+            Project randomProject = new ProjectDataGenerator().Generate();
```

Comment:
Don't generate a project in a highlightgenerator

Reply:
What should be the faker value for project then?

Reply:
just keep it null and set it in your test